source/includes/_embed2.md.erb

```diff
@@ -381,6 +383,17 @@ This action requires the `developer_proactive` feature flag to be turned on.
 
 Deletes the `chatter` used to fetch conversation logs for an end-user from storage. When a user opens a new Chat window a new `chatter` will be generated.
 
+#### evaluateCampaignConditions
+`evaluateCampaignConditions(options: object): Promise<void>;`
+
+This is similar to `triggerCampaign` - however, instead of triggering a specific Campaign, `evaluateCampaignConditions` evaluates the trigger conditions of the Campaigns in priority order and triggers the first Campaign (if any) whose conditions are matched.
```

Comment:
Maybe explain what you'd use this for?

Reply:
"This function can be used to test that campaign priorities and URL matches are working as you expect."

^ something like that?

Reply:
Liam raised a good point: Do we even want to expose that? Or is that an implementation detail?

Reply:
Expose what?

Reply:
I thought it's always been our plan to include it in the embed API, and currently it's implemented as being available for anyone with the ff - but if we don't want people to use it we could take it out of the docs